In [1]:
# Setup

import pandas as pd
from pathlib import Path
import hvplot.pandas
import requests
import json
from pprint import pprint
from config import geoapify_key

# Import and read files

file = Path("data/geoapify_df.csv")
df = pd.read_csv(file, encoding="utf-8")

In [2]:
# Create Country DF to show Number and Percent of Growing, Declining, Stagnant, and Total Startups for each Country

# Calculate number and percentages of startups

number_growing = []
number_declining = []
number_stagnant = []
number_total = []

unique_countries = df["Country"].unique()
countries_list = unique_countries.tolist()

for country in countries_list:
    growing = 0
    declining = 0
    stagnant = 0
    total = 0
    for index, row in df.iterrows():
        if row["Country"] == country:
            total = total + 1
            if row["CAGR (%)"] > 0:
                growing = growing + 1
            elif row["CAGR (%)"] < 0:
                declining = declining + 1
            else:
                stagnant = stagnant + 1
    number_growing.append(growing)
    number_declining.append(declining)
    number_stagnant.append(stagnant)
    number_total.append(total)

country_df = pd.DataFrame({
    "Country": countries_list,
    "Number of Growing Startups": number_growing,
    "Number of Declining Startups": number_declining,
    "Number of Stagnant Startups": number_stagnant,
    "Total Number of Startups": number_total
})

country_df["Percent of Growing Startups"] = (country_df["Number of Growing Startups"]/country_df["Total Number of Startups"]) * 100
country_df["Percent of Declining Startups"] = (country_df["Number of Declining Startups"]/country_df["Total Number of Startups"]) * 100
country_df["Percent of Stagnant Startups"] = (country_df["Number of Stagnant Startups"]/country_df["Total Number of Startups"]) * 100
country_df["Percent of World's Startups"] = (country_df["Total Number of Startups"]/1207) * 100

# Get Lat and Lng of each Country using Geoapify API

latitude = []
longitude = []

for index, row in country_df.iterrows():

    # Set search parameters

    target_country = f"{row['Country']}"

    # Base url

    base_url = f"https://api.geoapify.com/v1/geocode/search?country={target_country}&format=json&apiKey={geoapify_key}"

    # Run request and get longtitude and latitude
    
    response = requests.get(base_url).json()
    try:
        lat = response["results"][0]["lat"]
        latitude.append(lat)
    except:
        latitude.append(None)
        
    try:
        lng = response["results"][0]["lon"]
        longitude.append(lng)
    except:
        longitude.append(None)

country_df["Lat"] = latitude
country_df["Lng"] = longitude

country_df

Country  Number of Growing Startups  \
0                  China                           4   
1          United States                          48   
2              Australia                           2   
3         United Kingdom                           3   
4                  India                           7   
5                Germany                           2   
6                 Turkey                           1   
7             Seychelles                           0   
8                 Sweden                           0   
9                 Mexico                           0   
10               Estonia                           2   
11                Canada                           0   
12           Netherlands                           0   
13                France                           0   
14                Israel                           2   
15               Finland                           0   
16              Colombia                           0   
17               Belgium                           1   
18                Brazil                           0   
19           South Korea                           2   
20               Denmark                           0   
21           Switzerland                           0   
22             Lithuania                           1   
23               Austria                           0   
24               Ireland                           0   
25             Singapore                           2   
26             Indonesia                           0   
27               Vietnam                           0   
28             Hong Kong                           0   
29  United Arab Emirates                           0   
30             Argentina                           0   
31                 Spain                           0   
32                 Japan                           0   
33            Luxembourg                           0   
34               Nigeria                           0   
35               Croatia                           0   
36               Senegal                           0   
37              Malaysia                           0   
38               Bermuda                           0   
39                Norway                           0   
40          South Africa                           0   
41               Ecuador                           0   
42                 Chile                           0   
43              Thailand                           0   
44        Czech Republic                           0   
45           Philippines                           0   
46                 Italy                           0   

    Number of Declining Startups  Number of Stagnant Startups  \
0                              1                          144   
1                             15                          518   
2                              1                            5   
3                              3                           37   
4                              5                           47   
5                              1                           23   
6                              1                            1   
7                              0                            1   
8                              0                            5   
9                              0                            8   
10                             0                            3   
11                             0                           18   
12                             1                            6   
13                             2                           21   
14                             0                           16   
15                             0                            4   
16                             0                            3   
17                             0                            2   
18                             1                           15   
19                          

In [3]:
# Map visualization of Top 10 Countries with the greatest % of startups in the world

greatest_percent_startups = country_df.nlargest(10, "Percent of World's Startups")

# Plot Map

import holoviews as hv

map_greatest_percent_startups = greatest_percent_startups.hvplot.points("Lng", "Lat", geo=True, tiles="OSM", 
                                                                         size="Percent of World's Startups", color="Country", 
                                                                         width=1000, height=500, scale=10, alpha=0.8,
                                                                         hover_cols=["Percent of World's Startups"]).opts(
    title="Top 10 Countries with the Greatest Percent of All Unicorn Startups in the World")

# Save map

hv.save(map_greatest_percent_startups, "output_data/map_greatest_percent_startups.html")

map_greatest_percent_startups

:Overlay
   .WMTS.I   :WMTS   [Longitude,Latitude]
   .Points.I :Points   [Lng,Lat]   (Country,Percent of World's Startups)

In [5]:
# Map visualization of Top 10 Countries with the greatest # of startups in the world

greatest_number_startups = country_df.nlargest(10, "Total Number of Startups")

# Plot Map

map_greatest_number_startups = greatest_number_startups.hvplot.points("Lng", "Lat", geo=True, tiles="OSM", 
                                                                         size="Total Number of Startups", color="Country", 
                                                                         width=1000, height=500, scale=3, alpha=0.8,
                                                                         hover_cols=["Total Number of Startups"]).opts(
    title="Top 10 Countries with the Greatest Number of Unicorn Startups in the World")

# Save map

hv.save(map_greatest_number_startups, "output_data/map_greatest_number_startups.html")

map_greatest_number_startups

:Overlay
   .WMTS.I   :WMTS   [Longitude,Latitude]
   .Points.I :Points   [Lng,Lat]   (Country,Total Number of Startups)

In [6]:
# Map visualization of Top 10 Countries with the greatest # of growing startups and greatest # of declining startups

greatest_number_growing = country_df.nlargest(10, "Number of Growing Startups")
greatest_number_declining = country_df.nlargest(5, "Number of Declining Startups")

# Plot Map of Top 10 Countries with the greatest # of growing startups

map_greatest_number_growing = greatest_number_growing.hvplot.points("Lng", "Lat", geo=True, tiles="OSM", 
                                                                         size="Number of Growing Startups", color="green", 
                                                                         width=1000, height=500, scale=8, alpha=0.8,
                                                                         hover_cols=["Country", "Number of Growing Startups"])

# Plot Map of Top 5 Countries with the greatest # of declining startups

map_greatest_number_declining = greatest_number_declining.hvplot.points("Lng", "Lat", geo=True, tiles="OSM", 
                                                                         size="Number of Declining Startups", color="red", 
                                                                         width=1000, height=500, scale=8, alpha=0.8,
                                                                         hover_cols=["Country", "Number of Declining Startups"])

# Combine maps

map_greatest_number_growing_declining = map_greatest_number_growing * map_greatest_number_declining
map_greatest_number_growing_declining = map_greatest_number_growing_declining.opts(
    title="Top Countries with the Greatest Number of Growing or Declining Unicorn Startups")

# Save map

hv.save(map_greatest_number_growing_declining, "output_data/map_greatest_number_growing_declining.html")

map_greatest_number_growing_declining

:Overlay
   .WMTS.I    :WMTS   [Longitude,Latitude]
   .Points.I  :Points   [Lng,Lat]   (Number of Growing Startups,Country)
   .WMTS.II   :WMTS   [Longitude,Latitude]
   .Points.II :Points   [Lng,Lat]   (Number of Declining Startups,Country)

In [7]:
# Map visualization of Top 10 Countries with the greatest % of growing startups and greatest % of declining startups

greatest_percent_growing = country_df.nlargest(10, "Percent of Growing Startups")
greatest_percent_declining = country_df.nlargest(10, "Percent of Declining Startups")

# Plot Map of Top 10 Countries with the greatest % of growing startups

map_greatest_percent_growing = greatest_percent_growing.hvplot.points("Lng", "Lat", geo=True, tiles="OSM", 
                                                                         size="Percent of Growing Startups", color="green", 
                                                                         width=1000, height=500, scale=5, alpha=0.8,
                                                                         hover_cols=["Country", "Percent of Growing Startups"])

# Plot Map of Top 10 Countries with the greatest % of declining startups

map_greatest_percent_declining = greatest_percent_declining.hvplot.points("Lng", "Lat", geo=True, tiles="OSM", 
                                                                         size="Percent of Declining Startups", color="red", 
                                                                         width=1000, height=500, scale=5, alpha=0.8,
                                                                         hover_cols=["Country", "Percent of Declining Startups"])

# Combine maps

map_greatest_percent_growing_declining = map_greatest_percent_growing * map_greatest_percent_declining
map_greatest_percent_growing_declining = map_greatest_percent_growing_declining.opts(
    title="Top Countries with Greatest Percent of Growing or Declining Unicorn Startups\n(Percentages are calculated out of the total number of unicorn startups per country, not globally.)")

# Save map

hv.save(map_greatest_percent_growing_declining, "output_data/map_greatest_percent_growing_declining.html")

map_greatest_percent_growing_declining

:Overlay
   .WMTS.I    :WMTS   [Longitude,Latitude]
   .Points.I  :Points   [Lng,Lat]   (Percent of Growing Startups,Country)
   .WMTS.II   :WMTS   [Longitude,Latitude]
   .Points.II :Points   [Lng,Lat]   (Percent of Declining Startups,Country)

In [8]:
# Clean df

df = df.drop(["Unnamed: 0.1", "Unnamed: 0"], axis=1)
df = df.drop_duplicates(subset=["Company", "Date Joined"])

In [9]:
# Map visualization of Top 10 Startups with the greatest CAGR and lowest CAGR (by city)

top_startups = df.nlargest(10, "CAGR (%)")
bottom_startups = df.nsmallest(10, "CAGR (%)")

# Map plot

map_top_startups = top_startups.hvplot.points("Lng", "Lat", geo=True, tiles="OSM", 
                                              size="2024_Valuation", color="green", 
                                              width=1000, height=500, scale=5, alpha=0.5,
                                              hover_cols=["Company", "Country", "CAGR (%)"])

map_bottom_startups = bottom_startups.hvplot.points("Lng", "Lat", geo=True, tiles="OSM", 
                                                    size="2024_Valuation", color="red", 
                                                    width=1000, height=500, scale=5, alpha=0.5,
                                                    hover_cols=["Company", "Country", "CAGR (%)"])

# Combine maps

map_startups = map_top_startups * map_bottom_startups
map_startups = map_startups.opts(
    title="Top 10 Startups with the Greatest and Lowest CAGR")

# Save map

hv.save(map_startups, "output_data/map_top_startups.html")

map_startups

:Overlay
   .WMTS.I    :WMTS   [Longitude,Latitude]
   .Points.I  :Points   [Lng,Lat]   (2024_Valuation,Company,Country,CAGR (%))
   .WMTS.II   :WMTS   [Longitude,Latitude]
   .Points.II :Points   [Lng,Lat]   (2024_Valuation,Company,Country,CAGR (%))

In [4]:
# DF of Top 10 Countries with the greatest % of startups in the world

print("Top 10 Countries with the greatest % of startups in the world")

greatest_percent_startups

Top 10 Countries with the greatest % of startups in the world


Country  Number of Growing Startups  Number of Declining Startups  \
1    United States                          48                            15   
0            China                           4                             1   
4            India                           7                             5   
3   United Kingdom                           3                             3   
5          Germany                           2                             1   
13          France                           0                             2   
11          Canada                           0                             0   
14          Israel                           2                             0   
18          Brazil                           0                             1   
19     South Korea                           2                             0   

    Number of Stagnant Startups  Total Number of Startups  \
1                           518                       581   
0                           144                       149   
4                            47                        59   
3                            37                        43   
5                            23                        26   
13                           21                        23   
11                           18                        18   
14                           16                        18   
18                           15                        16   
19                           11                        13   

    Percent of Growing Startups  Percent of Declining Startups  \
1                      8.261618                       2.581756   
0                      2.684564                       0.671141   
4                     11.864407                       8.474576   
3                      6.976744                       6.976744   
5                      7.692308                       3.846154   
13                     0.000000                       8.695652   
11                     0.000000                       0.000000   
14                    11.111111                       0.000000   
18                     0.000000                       6.250000   
19                    15.384615                       0.000000   

    Percent of Stagnant Startups  Percent of World's Startups        Lat  \
1                      89.156627                    48.135874  39.783730   
0                      96.644295                    12.344656  35.000074   
4                      79.661017                     4.888152  22.351115   
3                      86.046512                     3.562552  54.702354   
5                      88.461538                     2.154101  51.163818   
13                     91.304348                     1.905551  46.603354   
11                    100.000000                     1.491301  61.066692   
14                     88.888889                     1.491301  30.812425   
18                     93.750000                     1.325601 -10.333333   
19                     84.615385                     1.077051  36.638392   

           Lng  
1  -100.445882  
0   104.999927  
4    78.667743  
3    -3.276575  
5    10.447831  
13    1.888334  
11 -107.991707  
14   34.859476  
18  -53.200000  
19  127.696119

In [12]:
# DF of Top 10 Countries with the greatest # of startups in the world

print("Top 10 Countries with the greatest # of startups in the world")

greatest_number_startups

Top 10 Countries with the greatest # of startups in the world


Country  Number of Growing Startups  Number of Declining Startups  \
1    United States                          48                            15   
0            China                           4                             1   
4            India                           7                             5   
3   United Kingdom                           3                             3   
5          Germany                           2                             1   
13          France                           0                             2   
11          Canada                           0                             0   
14          Israel                           2                             0   
18          Brazil                           0                             1   
19     South Korea                           2                             0   

    Number of Stagnant Startups  Total Number of Startups  \
1                           518                       581   
0                           144                       149   
4                            47                        59   
3                            37                        43   
5                            23                        26   
13                           21                        23   
11                           18                        18   
14                           16                        18   
18                           15                        16   
19                           11                        13   

    Percent of Growing Startups  Percent of Declining Startups  \
1                      8.261618                       2.581756   
0                      2.684564                       0.671141   
4                     11.864407                       8.474576   
3                      6.976744                       6.976744   
5                      7.692308                       3.846154   
13                     0.000000                       8.695652   
11                     0.000000                       0.000000   
14                    11.111111                       0.000000   
18                     0.000000                       6.250000   
19                    15.384615                       0.000000   

    Percent of Stagnant Startups  Percent of World's Startups        Lat  \
1                      89.156627                    48.135874  39.783730   
0                      96.644295                    12.344656  35.000074   
4                      79.661017                     4.888152  22.351115   
3                      86.046512                     3.562552  54.702354   
5                      88.461538                     2.154101  51.163818   
13                     91.304348                     1.905551  46.603354   
11                    100.000000                     1.491301  61.066692   
14                     88.888889                     1.491301  30.812425   
18                     93.750000                     1.325601 -10.333333   
19                     84.615385                     1.077051  36.638392   

           Lng  
1  -100.445882  
0   104.999927  
4    78.667743  
3    -3.276575  
5    10.447831  
13    1.888334  
11 -107.991707  
14   34.859476  
18  -53.200000  
19  127.696119

In [13]:
# DF of Top 10 Countries with the greatest # of growing startups

print("Top 10 Countries with the greatest # of growing startups")

greatest_number_growing

Top 10 Countries with the greatest # of growing startups


Country  Number of Growing Startups  Number of Declining Startups  \
1    United States                          48                            15   
4            India                           7                             5   
0            China                           4                             1   
3   United Kingdom                           3                             3   
2        Australia                           2                             1   
5          Germany                           2                             1   
10         Estonia                           2                             0   
14          Israel                           2                             0   
19     South Korea                           2                             0   
25       Singapore                           2                             0   

    Number of Stagnant Startups  Total Number of Startups  \
1                           518                       581   
4                            47                        59   
0                           144                       149   
3                            37                        43   
2                             5                         8   
5                            23                        26   
10                            3                         5   
14                           16                        18   
19                           11                        13   
25                           11                        13   

    Percent of Growing Startups  Percent of Declining Startups  \
1                      8.261618                       2.581756   
4                     11.864407                       8.474576   
0                      2.684564                       0.671141   
3                      6.976744                       6.976744   
2                     25.000000                      12.500000   
5                      7.692308                       3.846154   
10                    40.000000                       0.000000   
14                    11.111111                       0.000000   
19                    15.384615                       0.000000   
25                    15.384615                       0.000000   

    Percent of Stagnant Startups  Percent of World's Startups        Lat  \
1                      89.156627                    48.135874  39.783730   
4                      79.661017                     4.888152  22.351115   
0                      96.644295                    12.344656  35.000074   
3                      86.046512                     3.562552  54.702354   
2                      62.500000                     0.662800 -24.776109   
5                      88.461538                     2.154101  51.163818   
10                     60.000000                     0.414250  58.752378   
14                     88.888889                     1.491301  30.812425   
19                     84.615385                     1.077051  36.638392   
25                     84.615385                     1.077051   1.357107   

           Lng  
1  -100.445882  
4    78.667743  
0   104.999927  
3    -3.276575  
2   134.755000  
5    10.447831  
10   25.331908  
14   34.859476  
19  127.696119  
25  103.819499

In [15]:
# DF of Top 10 Countries with the greatest # of declining startups

print("Top 5 Countries with the greatest # of declining startups")

greatest_number_declining

Top 5 Countries with the greatest # of declining startups


Country  Number of Growing Startups  Number of Declining Startups  \
1    United States                          48                            15   
4            India                           7                             5   
3   United Kingdom                           3                             3   
13          France                           0                             2   
0            China                           4                             1   

    Number of Stagnant Startups  Total Number of Startups  \
1                           518                       581   
4                            47                        59   
3                            37                        43   
13                           21                        23   
0                           144                       149   

    Percent of Growing Startups  Percent of Declining Startups  \
1                      8.261618                       2.581756   
4                     11.864407                       8.474576   
3                      6.976744                       6.976744   
13                     0.000000                       8.695652   
0                      2.684564                       0.671141   

    Percent of Stagnant Startups  Percent of World's Startups        Lat  \
1                      89.156627                    48.135874  39.783730   
4                      79.661017                     4.888152  22.351115   
3                      86.046512                     3.562552  54.702354   
13                     91.304348                     1.905551  46.603354   
0                      96.644295                    12.344656  35.000074   

           Lng  
1  -100.445882  
4    78.667743  
3    -3.276575  
13    1.888334  
0   104.999927

In [16]:
# DF of Top 10 Countries with the greatest % of growing startups

print("Top 10 Countries with the greatest % of growing startups")

greatest_percent_growing

Top 10 Countries with the greatest % of growing startups


Country  Number of Growing Startups  Number of Declining Startups  \
22      Lithuania                           1                             0   
10        Estonia                           2                             0   
6          Turkey                           1                             1   
17        Belgium                           1                             0   
2       Australia                           2                             1   
19    South Korea                           2                             0   
25      Singapore                           2                             0   
4           India                           7                             5   
14         Israel                           2                             0   
1   United States                          48                            15   

    Number of Stagnant Startups  Total Number of Startups  \
22                            1                         2   
10                            3                         5   
6                             1                         3   
17                            2                         3   
2                             5                         8   
19                           11                        13   
25                           11                        13   
4                            47                        59   
14                           16                        18   
1                           518                       581   

    Percent of Growing Startups  Percent of Declining Startups  \
22                    50.000000                       0.000000   
10                    40.000000                       0.000000   
6                     33.333333                      33.333333   
17                    33.333333                       0.000000   
2                     25.000000                      12.500000   
19                    15.384615                       0.000000   
25                    15.384615                       0.000000   
4                     11.864407                       8.474576   
14                    11.111111                       0.000000   
1                      8.261618                       2.581756   

    Percent of Stagnant Startups  Percent of World's Startups        Lat  \
22                     50.000000                     0.165700  55.350000   
10                     60.000000                     0.414250  58.752378   
6                      33.333333                     0.248550  38.959759   
17                     66.666667                     0.248550  50.640281   
2                      62.500000                     0.662800 -24.776109   
19                     84.615385                     1.077051  36.638392   
25                     84.615385                     1.077051   1.357107   
4                      79.661017                     4.888152  22.351115   
14                     88.888889                     1.491301  30.812425   
1                      89.156627                    48.135874  39.783730   

           Lng  
22   23.750000  
10   25.331908  
6    34.924965  
17    4.666715  
2   134.755000  
19  127.696119  
25  103.819499  
4    78.667743  
14   34.859476  
1  -100.445882

In [17]:
# DF of Top 10 Countries with the greatest % of declining startups

print("Top 10 Countries with the greatest % of declining startups")

greatest_percent_declining

Top 10 Countries with the greatest % of declining startups


Country  Number of Growing Startups  Number of Declining Startups  \
6           Turkey                           1                             1   
12     Netherlands                           0                             1   
2        Australia                           2                             1   
13          France                           0                             2   
4            India                           7                             5   
3   United Kingdom                           3                             3   
18          Brazil                           0                             1   
5          Germany                           2                             1   
1    United States                          48                            15   
0            China                           4                             1   

    Number of Stagnant Startups  Total Number of Startups  \
6                             1                         3   
12                            6                         7   
2                             5                         8   
13                           21                        23   
4                            47                        59   
3                            37                        43   
18                           15                        16   
5                            23                        26   
1                           518                       581   
0                           144                       149   

    Percent of Growing Startups  Percent of Declining Startups  \
6                     33.333333                      33.333333   
12                     0.000000                      14.285714   
2                     25.000000                      12.500000   
13                     0.000000                       8.695652   
4                     11.864407                       8.474576   
3                      6.976744                       6.976744   
18                     0.000000                       6.250000   
5                      7.692308                       3.846154   
1                      8.261618                       2.581756   
0                      2.684564                       0.671141   

    Percent of Stagnant Startups  Percent of World's Startups        Lat  \
6                      33.333333                     0.248550  38.959759   
12                     85.714286                     0.579950  52.243498   
2                      62.500000                     0.662800 -24.776109   
13                     91.304348                     1.905551  46.603354   
4                      79.661017                     4.888152  22.351115   
3                      86.046512                     3.562552  54.702354   
18                     93.750000                     1.325601 -10.333333   
5                      88.461538                     2.154101  51.163818   
1                      89.156627                    48.135874  39.783730   
0                      96.644295                    12.344656  35.000074   

           Lng  
6    34.924965  
12    5.634323  
2   134.755000  
13    1.888334  
4    78.667743  
3    -3.276575  
18  -53.200000  
5    10.447831  
1  -100.445882  
0   104.999927

In [20]:
# DF of Top 10 Startups with the greatest CAGR

print("Top 10 Startups with the greatest CAGR")

top_startups

Top 10 Startups with the greatest CAGR


Company  2022_Valuation  2023_Valuation  2024_Valuation  \
299           OpenAI            2.92           29.00           80.00   
819           Branch            1.05            4.00            4.00   
403       Improbable            1.00            3.36            3.36   
1048  LayerZero Labs            1.00            3.00            3.00   
952        Shield AI            1.00            2.30            2.80   
933          Claroty            1.00            1.00            2.50   
206        VAST Data            3.70            3.70            9.10   
460     Hugging Face            2.00            2.00            4.50   
899           Skydio            1.00            2.20            2.20   
1061           Glean            1.00            1.00            2.20   

     Date Joined         Country           City  \
299    7/22/2019   United States  San Francisco   
819     3/1/2022   United States       Columbus   
403    5/11/2017  United Kingdom         London   
1048   3/30/2022   United States       New York   
952    8/24/2021   United States      San Diego   
933    6/17/2021   United States       New York   
206    4/16/2020   United States       New York   
460     5/9/2022   United States       New York   
899    2/25/2021   United States   Redwood City   
1061   5/18/2022   United States      Palo Alto   

                                 Industry  \
299               Artificial intelligence   
819                               Fintech   
403                                 Other   
1048         Internet software & services   
952               Artificial intelligence   
933                         Cybersecurity   
206           Data management & analytics   
460               Artificial intelligence   
899   Supply chain, logistics, & delivery   
1061         Internet software & services   

                                       Select Investors  Overall_Growth  \
299                                     Khosla Ventures           77.08   
819     American Family Ventures, Greycroft, SignalFire            2.95   
403   Andreessen Horowitz, SoftBank Group, Temasek H...            2.36   
1048  Andreessen Horowitz, FTX Ventures, Tiger Globa...            2.00   
952     Andreessen Horowitz, Homebrew, Point72 Ventures            1.80   
933            Bessemer Venture Partners, MoreVC, Team8            1.50   
206   Norwest Venture Partners, Goldman Sachs, Dell ...            5.40   
460           Betaworks Ventures, Addition, Lux Capital            2.50   
899   Andreessen Horowitz, Andreessen Horowitz, Inst...            1.20   
1061  General Catalyst, Kleiner Perkins Caufield & B...            1.20   

      CAGR (%)        Lat         Lng  
299     423.42  37.640314 -123.173825  
819      95.18  39.808694  -83.210180  
403      83.30  51.286760   -0.510375  
1048     73.21  40.476578  -74.258843  
952      67.33  32.534798 -117.309816  
933      58.11  40.476578  -74.258843  
206      56.83  40.476578  -74.258843  
460      50.00  40.476578  -74.258843  
899      48.32  37.447172 -122.290823  
1061     48.32  37.287679 -122.202665

In [21]:
# DF of Top 10 Startups with the lowest CAGR

print("Top 10 Startups with the lowest CAGR")

bottom_startups

Top 10 Startups with the lowest CAGR


Company  2022_Valuation  2023_Valuation  2024_Valuation  \
5      Checkout.com           40.00           11.00           11.00   
180         Patreon            4.00            1.35            1.35   
304           Flink            2.85            1.07            1.07   
48       OutSystems            9.50            4.30            4.30   
19   Blockchain.com           14.00           14.00            7.00   
301            Loft            2.90            2.90            1.46   
11           BYJU's           22.00           11.50           11.50   
260           Udaan            3.10            3.10            1.70   
4             Canva           40.00           40.00           25.40   
2             SHEIN          100.00           66.00           66.00   

    Date Joined         Country           City  \
5      5/2/2019  United Kingdom         London   
180    9/2/2020   United States  San Francisco   
304   12/1/2021         Germany         Berlin   
48     6/5/2018   United States         Boston   
19    2/17/2021  United Kingdom         London   
301    1/3/2020          Brazil      Sao Paulo   
11    7/25/2017           India      Bengaluru   
260    9/4/2018           India      Bengaluru   
4      1/8/2018       Australia    Surry Hills   
2      7/3/2018           China       Shenzhen   

                                Industry  \
5                                Fintech   
180         Internet software & services   
304      E-commerce & direct-to-consumer   
48          Internet software & services   
19                               Fintech   
301      E-commerce & direct-to-consumer   
11                                Edtech   
260  Supply chain, logistics, & delivery   
4           Internet software & services   
2        E-commerce & direct-to-consumer   

                                      Select Investors  Overall_Growth  \
5    Tiger Global Management, Insight Partners, DST...          -29.00   
180                Index Ventures, Thrive Capital, CRV           -2.65   
304            Mubadala Capital, Bond, Prosus Ventures           -1.78   
48        KKR, ES Ventures, North Bridge Growth Equity           -5.20   
19   Lightspeed Venture Partners, Google Ventures, ...           -7.00   
301     Monashees+, Andreessen Horowitz, QED Investors           -1.44   
11   Tencent Holdings, Lightspeed India Partners, S...          -10.50   
260  DST Global, Lightspeed Venture Partners, Micro...           -1.40   
4    Sequoia Capital China, Blackbird Ventures, Mat...          -14.60   
2    Tiger Global Management, Sequoia Capital China...          -34.00   

     CAGR (%)        Lat         Lng  
5      -47.56  51.286760   -0.510375  
180    -41.91  37.640314 -123.173825  
304    -38.73  52.338245   13.088345  
48     -32.72  42.227911  -71.191244  
19     -29.29  51.286760   -0.510375  
301    -29.05 -24.007900  -46.826269  
11     -27.70  12.816794   77.430082  
260    -25.95  12.816794   77.430082  
4      -20.31 -33.892106  151.201846  
2      -18.76  22.287600  113.679356

In [6]:
country_df["Ratio of Growing to Declining"] = country_df["Number of Growing Startups"]/(country_df["Number of Declining Startups"] + country_df["Number of Stagnant Startups"])

country_df

Country  Number of Growing Startups  \
0                  China                           4   
1          United States                          48   
2              Australia                           2   
3         United Kingdom                           3   
4                  India                           7   
5                Germany                           2   
6                 Turkey                           1   
7             Seychelles                           0   
8                 Sweden                           0   
9                 Mexico                           0   
10               Estonia                           2   
11                Canada                           0   
12           Netherlands                           0   
13                France                           0   
14                Israel                           2   
15               Finland                           0   
16              Colombia                           0   
17               Belgium                           1   
18                Brazil                           0   
19           South Korea                           2   
20               Denmark                           0   
21           Switzerland                           0   
22             Lithuania                           1   
23               Austria                           0   
24               Ireland                           0   
25             Singapore                           2   
26             Indonesia                           0   
27               Vietnam                           0   
28             Hong Kong                           0   
29  United Arab Emirates                           0   
30             Argentina                           0   
31                 Spain                           0   
32                 Japan                           0   
33            Luxembourg                           0   
34               Nigeria                           0   
35               Croatia                           0   
36               Senegal                           0   
37              Malaysia                           0   
38               Bermuda                           0   
39                Norway                           0   
40          South Africa                           0   
41               Ecuador                           0   
42                 Chile                           0   
43              Thailand                           0   
44        Czech Republic                           0   
45           Philippines                           0   
46                 Italy                           0   

    Number of Declining Startups  Number of Stagnant Startups  \
0                              1                          144   
1                             15                          518   
2                              1                            5   
3                              3                           37   
4                              5                           47   
5                              1                           23   
6                              1                            1   
7                              0                            1   
8                              0                            5   
9                              0                            8   
10                             0                            3   
11                             0                           18   
12                             1                            6   
13                             2                           21   
14                             0                           16   
15                             0                            4   
16                             0                            3   
17                             0                            2   
18                             1                           15   
19                          

Analysis of Map Visualizations

These maps collectively highlight the global distribution and trends of unicorn startups. This could provide valuable insights for investors interested in the startup landscape. 

It is clear that the United States and China dominate the startup ecosystem with the largest share and number of unicorn startups. The U.S. contains 43.14% of the world's unicorn startups, while China contains 12.34%. 

Countries with a larger volume of startups, including the U.S., China, and India, demonstrate greater stability in terms of the growth and decline of unicorn startups. Roughly 8.26% of U.S. startups are growing and 2.58% are declining; 2.68% of startups in China are growing while 0.67% are declining. 11.86% of startups in India are growing and 8.47% are declining. Conversely, these numbers are much higher in countries with smaller startup ecosystems. They tend to have either a higher percentage of growing startups or a higher percentage of declining startups within their respective countries. This suggests that countries with a smaller startup ecosystem experience greater volatility and less stability. Infrastructure and political and economic conditions may have a more pronounced impact on whether these countries experience high-growth or high-risk opportunities. 